ztp/gitops-subscriptions/argocd/SNOExpansion.md

```diff
@@ -0,0 +1,483 @@
+# SNO expansion #
+
+SNO expansion is a scenario where initially a SNO is deployed into production, and at a later date the cluster size is increased by the addition of a worker node.
+The resulting cluster will contain two nodes:
+- Original SNO serving as the control plane for itself and the additional node
+- One additional node without any control plane components (controlled by the SNO).
+
+This transition incurs zero downtime on the original SNO node.
+Addition of more than one worker node to the SNO cluster is possible, but has not been verified in this version.
+
+## Prerequisites ##
+1. A cluster installed and configured using the GitOps ZTP fow, as described in [GitOps ZTP pipeline](README.md)
+1. ACM 2.6 (or above) with MultiClusterHub created and configured, running on OCP 4.11 (or above) bare metal cluster
```

Comment:
Why wouldn't this work with 2.5 ?

Reply:
According to my understanding of [this](https://coreos.slack.com/archives/CUPJTHQ5P/p1653476931714459) this code of central infra management is going to be included only in 2.6.